en/docs/setup/setting-up-deployment-pattern-1.md

```diff
@@ -373,72 +319,64 @@ Follow the instructions below to cluster WSO2 Identity Server.
-                range, the faster the time it takes to discover members since
-                each node has to scan a lesser number of potential members.
+            You can also use IP address ranges for the
+            `              hostName             ` . For example,
+            192.168.1.2-10. This should ensure that the cluster eventually
+            recovers after failures. One shortcoming of doing this is that
+            you can define a range only for the last portion of the IP
+            address. You should also keep in mind that the smaller the
+            range, the faster the time it takes to discover members since
+            each node has to scan a lesser number of potential members.
         
                   
         
 
 3.  All the caches are considered to be local caches in Identity Server
     by default. However, for clustered nodes enable local cache
-    invalidation by setting the **ForceLocalCache** property within the
-    `            <cache>           ` section in the
-    `            carbon.xml           ` file in
+    invalidation by setting the `fource_local_cache` property in the
+    `            deployment.toml           ` file in
     `            <IS_HOME>/repository/conf/           ` to
     `            true           ` .
 
     ``` java
-    <ForceLocalCache>true</ForceLocalCache>
+    [server]
+    force_local_cache="true"
     ```
 
 4.  Configure the following.
     1.  Change the datasource name to
-        `              jdbc/WSO2UMDB             ` in **user-mgt.xml**
+        `              jdbc/WSO2UMDB             ` in `user-mgt.xml`
         (located in
         `              <IS_HOME>/repository/conf/             ` ). This
         refers to the user store you configured in the [Configuring the
         user
-        store](https://docs.wso2.com/display/IS541/Setting+Up+Deployment+Pattern+1#SettingUpDeploymentPattern1-Configuringtheuserstore)
+        store](#configuring-the-user-store)
         section above.
 
-        **user-mgt.xml**
-
-        ``` html/xml
-                <UserManager>
-                  <Realm>
-                  <Configuration>
-                  ...
-                  <Property name="dataSource">jdbc/WSO2CarbonDB</Property>
-                  </Configuration>
-                  ...
-                  </Realm>
-                </UserManager>
+        ``` xml tab="user-mgt.xml"
+        <UserManager>
+            <Realm>
+            <Configuration>
+            ...
+            <Property name="dataSource">jdbc/WSO2CarbonDB</Property>
+            </Configuration>
+            ...
+            </Realm>
+        </UserManager>
         ```
 
     2.  Change the datasource name to
         `              jdbc/WSO2IDENTITYDB             ` in
-        **identity.xml** (located in
-        `              <IS_HOME>/repository/conf/identity             `
+        `deployment.clustering` (located in
```

Comment:
is there a deployment.clustering file or is it deployment.toml?

Reply:
The respective file was removed from the commit as the file is subject to change—it will be reflected in a subsequent PR.